<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta name="generator" content="quarto-1.0.36">
<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">
<meta name="author" content="Jinseob Kim">
<meta name="dcterms.date" content="2018-11-08">
<meta name="description" content="유전율(heritability)은 어떤 형질의 유전적인 측면을 정량적으로 설명하는 유용한 지표이나 이분형 형질의 경우 해석이 어렵습니다. 이 때는 Sibling recurrence risk가 직관적인 지표이나, 유병률 정보가 필요하고 다른 변수의 보정이 어려운 문제가 있습니다. 본 연구에서는 흔히 쓰는 OR scale을 이용, 이분형 변수에서 직관적이고 다른 변수의 보정도 가능한 유전율 지표를 제안합니다.">
<title>Zarathu Blog - New Scale Measure of Heritability in Binary Trait Using Median Odds Ratio: Median OR Ratio</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
span.underline{text-decoration: underline;}
div.column{display: inline-block; vertical-align: top; width: 50%;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

<script src="../../site_libs/quarto-nav/quarto-nav.js"></script>
<script src="../../site_libs/quarto-nav/headroom.min.js"></script>
<script src="../../site_libs/clipboard/clipboard.min.js"></script>
<script src="../../site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="../../site_libs/quarto-search/fuse.min.js"></script>
<script src="../../site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="../../">
<link href="../../img/logo_favicon.png" rel="icon" type="image/png">
<script src="../../site_libs/quarto-html/quarto.js"></script>
<script src="../../site_libs/quarto-html/popper.min.js"></script>
<script src="../../site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="../../site_libs/quarto-html/anchor.min.js"></script>
<link href="../../site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="../../site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="../../site_libs/bootstrap/bootstrap.min.js"></script>
<link href="../../site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="../../site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light"><script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 20,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit"
  }
}</script><script type="text/javascript">

(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
ga('create', 'UA-135478021-2', 'auto');

ga('send', {
  hitType: 'pageview',
  'anonymizeIp': true,
});
</script><style>html{ scroll-behavior: smooth; }</style>
<script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml-full.js" type="text/javascript"></script><link rel="stylesheet" href="../../styles.css">
</head>
<body class="floating nav-fixed">


<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top"><nav class="navbar navbar-expand-lg navbar-dark "><div class="navbar-container container-fluid">
      <a class="navbar-brand" href="../../index.html">
    <img src="../../img/logo.png" alt=""><span class="navbar-title">Zarathu Blog</span>
  </a>
          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarCollapse" aria-controls="navbarCollapse" aria-expanded="false" aria-label="Toggle navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
  <span class="navbar-toggler-icon"></span>
</button>
          <div class="collapse navbar-collapse" id="navbarCollapse">
            <ul class="navbar-nav navbar-nav-scroll ms-auto">
<li class="nav-item">
    <a class="nav-link" href="../../index.html">Home</a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="http://app.zarathu.com/">Applications</a>
  </li>  
  <li class="nav-item dropdown ">
    <a class="nav-link dropdown-toggle" href="#" id="nav-menu-r-packages" role="button" data-bs-toggle="dropdown" aria-expanded="false">R packages</a>
    <ul class="dropdown-menu dropdown-menu-end" aria-labelledby="nav-menu-r-packages">
<li>
    <a class="dropdown-item" href="https://jinseob2kim.github.io/jstable/">
 <span class="dropdown-text">jstable</span></a>
  </li>  
        <li>
    <a class="dropdown-item" href="https://jinseob2kim.github.io/jskm/">
 <span class="dropdown-text">jskm</span></a>
  </li>  
        <li>
    <a class="dropdown-item" href="https://jinseob2kim.github.io/jsmodule/">
 <span class="dropdown-text">jsmodule</span></a>
  </li>  
    </ul>
</li>
  <li class="nav-item">
    <a class="nav-link" href="../../contributors.html">Contributors</a>
  </li>  
  <li class="nav-item compact">
    <a class="nav-link" href="https://github.com/zarathucorp"><i class="bi bi-github" role="img">
</i> 
 </a>
  </li>  
  <li class="nav-item compact">
    <a class="nav-link" href="../../index.xml"><i class="bi bi-rss" role="img" aria-label="RSS">
</i> 
 </a>
  </li>  
</ul>
<div id="quarto-search" class="" title="Search"></div>
          </div> <!-- /navcollapse -->
      </div> <!-- /container-fluid -->
    </nav></header><!-- content --><header id="title-block-header" class="quarto-title-block default toc-left page-columns page-full"><div class="quarto-title-banner page-columns page-full">
    <div class="quarto-title column-body">
      <h1 class="title">New Scale Measure of Heritability in Binary Trait Using Median Odds Ratio: Median OR Ratio</h1>
                  <div>
        <div class="description">
          <p>유전율(heritability)은 어떤 형질의 유전적인 측면을 정량적으로 설명하는 유용한 지표이나 이분형 형질의 경우 해석이 어렵습니다. 이 때는 Sibling recurrence risk가 직관적인 지표이나, 유병률 정보가 필요하고 다른 변수의 보정이 어려운 문제가 있습니다. 본 연구에서는 흔히 쓰는 OR scale을 이용, 이분형 변수에서 직관적이고 다른 변수의 보정도 가능한 유전율 지표를 제안합니다.</p>
        </div>
      </div>
                          <div class="quarto-categories">
                <div class="quarto-category">article</div>
                <div class="quarto-category">statistics</div>
                <div class="quarto-category">R</div>
              </div>
                  </div>
  </div>
    
  <div class="quarto-title-meta-author">
    <div class="quarto-title-meta-heading">Author</div>
    <div class="quarto-title-meta-heading">Affiliation</div>
    
      <div class="quarto-title-meta-contents">
      <a href="https://github.com/jinseob2kim">Jinseob Kim</a> 
    </div>
      <div class="quarto-title-meta-contents">
          <p class="affiliation">
              ANPANMAN Co.,Ltd
            </p>
        </div>
      </div>

  <div class="quarto-title-meta">

        
      <div>
      <div class="quarto-title-meta-heading">Published</div>
      <div class="quarto-title-meta-contents">
        <p class="date">November 8, 2018</p>
      </div>
    </div>
      
    </div>
    
  
  </header><div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article page-navbar">
<!-- sidebar -->
  <nav id="quarto-sidebar" class="sidebar collapse sidebar-navigation floating overflow-auto"><nav id="TOC" role="doc-toc"><h2 id="toc-title">On this page</h2>
   
  <ul>
<li><a href="#abstract" id="toc-abstract" class="nav-link active" data-scroll-target="#abstract">Abstract</a></li>
  <li>
<a href="#introduction" id="toc-introduction" class="nav-link" data-scroll-target="#introduction">Introduction</a>
  <ul class="collapse">
<li><a href="#method" id="toc-method" class="nav-link" data-scroll-target="#method">Method</a></li>
  <li><a href="#breif-review-of-meidan-or" id="toc-breif-review-of-meidan-or" class="nav-link" data-scroll-target="#breif-review-of-meidan-or">Breif review of Meidan OR</a></li>
  <li><a href="#formulas-of-textmorr_su" id="toc-formulas-of-textmorr_su" class="nav-link" data-scroll-target="#formulas-of-textmorr_su">Formula’s of <span class="math inline">\(\text{MORR}_{SU}\)</span></a></li>
  </ul>
</li>
  <li><a href="#apply-to-real-data" id="toc-apply-to-real-data" class="nav-link" data-scroll-target="#apply-to-real-data">Apply to Real Data</a></li>
  <li><a href="#conclusion" id="toc-conclusion" class="nav-link" data-scroll-target="#conclusion">Conclusion</a></li>
  <li>
<a href="#appendix" id="toc-appendix" class="nav-link" data-scroll-target="#appendix">Appendix</a>
  <ul class="collapse">
<li><a href="#calculate-morr" id="toc-calculate-morr" class="nav-link" data-scroll-target="#calculate-morr">Calculate MORR</a></li>
  <li><a href="#compare-to-other-measures" id="toc-compare-to-other-measures" class="nav-link" data-scroll-target="#compare-to-other-measures">Compare to Other Measures</a></li>
  </ul>
</li>
  </ul></nav>
</nav><!-- margin-sidebar --><div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
    </div>
<!-- main -->
<main class="content quarto-banner-title-block" id="quarto-document-content"><p>본 연구는 <a href="https://github.com/jinseob2kim">김진섭</a> 대표가 계획했던 연구로, 결과적으로 학술지 게재에 실패했다는 것을 미리 알려드립니다.</p>
<section id="abstract" class="level2"><h2 class="anchored" data-anchor-id="abstract">Abstract</h2>
<p>Heritability는 trait의 유전적인 측면을 정량적으로 설명하는 지표로 genetic study에서 흔히 쓰이는 지표이다. <span class="math inline">\(Y\)</span>가 continuous variable일 때는 Intraclass Correlation Coeffcient(ICC) 혹은 Variance Partition Coefficients(VPC)의 형태로 heritability를 간단히 표시할 수 있으나, binary trait에는 이를 그대로 적용할 수 없다. 이에 liability threshold model 등의 다양한 approximation 방법이 이용되고 있으나 결과 해석의 어려움이 문제로 지적되고 있다. 한편 binary trait의 경우 sibling recurrence risk ratio(<span class="math inline">\(\lambda_s\)</span>)가 유전적인 부분을 표현하는 직관적인 지표로 쓰이고 있으나 prevalence값이 필요하고 covariate 보정이 안되는 한계점이 있다. 이에 저자는 binary multilevel study에서 이용되는 Median Odds Ratio(MOR)을 이용하여 binary trait의 유전적인 정도를 평가하는 편하고 직관적인 지표인 Median OR Ratio(<span class="math inline">\(\text{MORR}_{SU}\)</span>)을 제안하였으며 Healthy Twin Study, Korea의 hypertriglycemia trait에 적용하여 간단하며 직관적으로 유전적인 정도를 측정할 수 있었다. 이 지표가 binary trait에서 유전적인 정도를 표현하는 새로운 지표로서 heritability나 <span class="math inline">\(\lambda_s\)</span>를 보완할 수 있을 것으로 확신한다.</p>
</section><section id="introduction" class="level2"><h2 class="anchored" data-anchor-id="introduction">Introduction</h2>
<p>Heritability는 continuous trait에서 polygenic effect가 설명할 수 있는 정도를 측정하는 지표로 전체 분산 중 polygenic effect가 설명하는 비율(the portion of phenotypic variance in a population attributable to additive genetic factors)로 정의되며, trait의 정규분포 가정이 깨지지 않는다면 mixed effect model 등을 이용하여 쉽게 구할 수 있다.<span class="citation" data-cites="manolio2009finding vattikuti2012heritability">(<a href="#ref-manolio2009finding" role="doc-biblioref">Manolio et al. 2009</a>; <a href="#ref-vattikuti2012heritability" role="doc-biblioref">Vattikuti 2012</a>)</span>. Mixed effect model의 결과에서 polygenic effect의 분산과 residual의 분산이 추정되고 이를 통해 전체분산과 polygenic effect가 설명하는 분율을 계산하는 것인데 이는 Intraclass Correlation Coeffcient(ICC) 또는 Variance Partition Coefficients(VPC)의 형태이며, mixed effect model을 이용함으로서 covariate(age, sex, etc..)들을 보정할 수 있는 장점이 있다.</p>
<p>그러나 binary trait일 때는 이 방법으로 heritability를 계산할 수가 없다. 보통 질병에 걸릴 확률 <span class="math inline">\(p\)</span>가 Logistic distribution을 따른다고 가정하고 logistic regression model을 이용하게 되는데 이 때, Continuous variable에서 한 것처럼 추정된 variance of polygenic effect의 분산은 probability의 logit function의 scale에서 계산된 것이며, 전체분산은 probability scale에서 계산이 되기 때문에 이를 단순하게 분자, 분모로 하여 계산할 수 없다<span class="citation" data-cites="browne2005variance">(<a href="#ref-browne2005variance" role="doc-biblioref">Browne et al. 2005</a>)</span>. binary trait에서 heritability를 구하는 방법으로는 first order Taylor series expansion을 이용하여 logit함수에 대한 회귀식을 <span class="math inline">\(Y\)</span>에 대한 회귀식으로 approximation하여 linearisation 하는 방법, simulation을 이용하는 방법, latent variable을 이용하는 방법 등이 제시되어 있는데 이들 방법은 근본적으로 근사값의 계산이라는 한계점이 있으며, 해석을 어떻게 해야 되는지도 문제가 된다<span class="citation" data-cites="browne2005variance 1611.02910 vigre2004intr Davies2639 Tenesa2013">(<a href="#ref-browne2005variance" role="doc-biblioref">Browne et al. 2005</a>; <a href="#ref-1611.02910" role="doc-biblioref">Bonnet 2016</a>; <a href="#ref-vigre2004intr" role="doc-biblioref">Vigre et al. 2004</a>; <a href="#ref-Davies2639" role="doc-biblioref">Davies et al. 2015</a>; <a href="#ref-Tenesa2013" role="doc-biblioref">Tenesa and Haley 2013</a>)</span>.</p>
<p>binary trait에서 유전적인 정도를 표현하는 또다른 방법으로 recurrence risk ratio라는 것이 있는데, 이 중 흔히 이용되는 것은 sibling recurrence risk ratio(<span class="math inline">\(\lambda_s\)</span>)로 일반 인구집단의 prevalence에 비해 affected people의 sibling 집단에서의 prevalence가 몇배나 더 높은지를 표현하며 직관적으로 이해할 수 있는 지표라는 장점이 있다<span class="citation" data-cites="rybicki2000relationship">(<a href="#ref-rybicki2000relationship" role="doc-biblioref">Rybicki and Elston 2000</a>)</span>. 그러나 이 값은 일반 인구 집단에서의 prevalence를 정확히 측정해야 한다는 부담을 갖고 있으며 ascertainment bias에 민감하며, polygenic effect의 변화없이 common environmental factor의 영향에 의해서도 변화할 수 있다는 문제점이 있다<span class="citation" data-cites="guo1998inflation guo2000familial guo2002sibling">(<a href="#ref-guo1998inflation" role="doc-biblioref">S.-W. Guo 1998</a>, <a href="#ref-guo2002sibling" role="doc-biblioref">2002</a>; <a href="#ref-guo2000familial" role="doc-biblioref">S.-W. Guo 2000</a>)</span>.</p>
<p>한편 binary trait의 multilevel analysis에서 group level의 효과를 쉽게 이해하기 위해 Median Odds Ratio(MOR)라는 개념이 제시되었는데, 이것의 정의는 “the median value of the odds ratio between the group at highest risk and the group at lowest risk when randomly picking out two groups”이다<span class="citation" data-cites="merlo2006brief">(<a href="#ref-merlo2006brief" role="doc-biblioref">Merlo et al. 2006</a>)</span>. MOR은 group변수의 분산만을 이용하여 쉽게 계산될 수 있으며 OR scale이기 때문에 이해하기도 쉬운데, 1보다 클수록 group effect가 크고 1에 가까울수록 group effect가 없다고 해석할 수 있다<span class="citation" data-cites="larsen2000interpreting larsen2005appropriate merlo2006brief">(<a href="#ref-larsen2000interpreting" role="doc-biblioref">Larsen et al. 2000</a>; <a href="#ref-larsen2005appropriate" role="doc-biblioref">Larsen and Merlo 2005</a>; <a href="#ref-merlo2006brief" role="doc-biblioref">Merlo et al. 2006</a>)</span>.</p>
<p>이에 저자는 multilevel logistic regression에서 이용하였던 MOR의 개념을 이용하여 binary trait의 유전적인 정도를 측정하는 새로운 지표인 Median OR Ratio between Sibling Pairs and Unrelated Pairs(<span class="math inline">\(\text{MORR}_{SU}\)</span>)을 소개할 것이다. 본 지표는 sibling pairs끼리의 OR에 비해 unrelated pairs끼리의 OR의 값이 대략적으로(median) 얼마나 높은가를 나타내며, 유전적인 부분이 전혀 없는 질병이라면 sibling pair에서나 unrelated pair에서의 OR이 같게 되어 <span class="math inline">\(\text{MORR}_{SU}\)</span>의 값은 1이 된다. 반면에 이 값이 크면 클수록 sibling pair끼리의 질병발생 양상이 unrelated pair끼리의 그것보다 비슷해지고 이는 유전적인 부분이 큰 질병임을 뜻한다. 본 지표는 hierarchial generalized linear model(HGLM)를 통하여 계산할 수 있으며, covariates를 보정할 수 있다는 heritability의 장점과 쉽게 해석되는 <span class="math inline">\(\lambda_s\)</span>의 장점을 동시에 갖고 있다. 반대로 말하면 직관적인 해석이 어려운 heritability의 단점과 prevalence 정보가 필요하며 covariate보정이 어려운 <span class="math inline">\(\lambda_s\)</span>의 단점을 보완하였다고도 할 수 있다. <span class="citation" data-cites="lee1996hierarchical">(<a href="#ref-lee1996hierarchical" role="doc-biblioref">Lee and Nelder 1996</a>)</span>. 본 연구에서는 <span class="math inline">\(\text{MORR}_{SU}\)</span>의 개념을 소개한 후 실제 데이터인 Healthy twin study, Korea의 hypertriglycemia trait에 적용하여 유전적인 부분을 해석해 볼 것이다<span class="citation" data-cites="sung2006we">(<a href="#ref-sung2006we" role="doc-biblioref">SUNG et al. 2006</a>)</span>.</p>
<section id="method" class="level3"><h3 class="anchored" data-anchor-id="method">Method</h3>
</section><section id="breif-review-of-meidan-or" class="level3"><h3 class="anchored" data-anchor-id="breif-review-of-meidan-or">Breif review of Meidan OR</h3>
<p>먼저 MOR에 대해 간략하게 요약하여 설명하겠다<span class="citation" data-cites="larsen2000interpreting">(<a href="#ref-larsen2000interpreting" role="doc-biblioref">Larsen et al. 2000</a>)</span>.</p>
<p><span class="math inline">\(Y_{ij}\)</span>를 <span class="math inline">\(j\)</span>th group의 <span class="math inline">\(i\)</span>th individual의 health status라 하고(case: 1, control: 0), <span class="math inline">\(X_{ij}\)</span>를 vector of covariates, <span class="math inline">\(G_j\)</span>를 <span class="math inline">\(j\)</span>th group의 effect라 정의하자. 이 때 multilevel logistic regression의 formula를 mixed effect model로 표시하면 다음과 같다.</p>
<p><span class="math display">\[
\begin{aligned}
\text{Logit}[Pr(Y_{ij}=1|X_{ij},G_j)]=\beta_0+X_{ij}'\beta_1+G_j
\end{aligned}
\]</span></p>
<p>(<span class="math inline">\(\beta_0\)</span>: intercept, <span class="math inline">\(\beta_1\)</span>: vector of fixed regression coefficients, <span class="math inline">\(G_j\)</span>: random intercept <span class="math inline">\(G_j\sim \text{iid  } N(0,V_g)\)</span>)</p>
<p>이 때 Conditional Odds는</p>
<p><span class="math display">\[
\begin{aligned}
\text{Odds}[Pr(Y_{ij}=1|X_{ij},G_j)]=\exp{(\beta_0)}\exp{(X_{ij}'\beta_1)}\exp{(G_j)}
\end{aligned}
\]</span></p>
<p>로 표현할 수 있으며 X가 고정되어 있을 때 임의로 뽑은 <span class="math inline">\(j\)</span>th Group과 <span class="math inline">\(k\)</span>th Group의 Odds ratio는</p>
<p><span class="math display">\[
\begin{aligned}
\frac{\text{Odds}[Pr(Y_{ij}=1|X,G_j)]}{{\text{Odds}[Pr(Y_{ik}=1|X,G_k)]}}=\exp{(G_j-G_k)}
\end{aligned}
\]</span></p>
<p>가 되고 Odds가 큰그룹을 Odds가 작은 그룹과 비교한다면 <span class="math inline">\(\text{OR}=\exp{|G_j-G_k|}\)</span>이다. 이제 <span class="math inline">\((G_j-G_k)\sim N(0,2V_g)\)</span> 임을 이용하여 이것의 중앙값(median)을 계산하면</p>
<p><span class="math display">\[
\begin{aligned}
\text{MOR} = \exp{(\sqrt{2V_g}\times \Phi^{-1}{(0.75)})}\simeq \exp{(0.95\sqrt{V_g})}
\end{aligned}
\]</span></p>
<p>이 되고 이를 MOR로 정의한다. MOR은 VPC와 다르게 오직 group variable의 분산인 <span class="math inline">\(V_g\)</span>만 가지고 계산될 수 있어 그 자체값과 95% 신뢰구간을 간단히 계산할 수 있으며, MOR<span class="math inline">\(=1\)</span> 이라면 group variable의 effect가 없다고 해석할 수 있고 MOR이 커질수록 group variable의 effect가 크다고 해석할 수 있다.</p>
</section><section id="formulas-of-textmorr_su" class="level3"><h3 class="anchored" data-anchor-id="formulas-of-textmorr_su">Formula’s of <span class="math inline">\(\text{MORR}_{SU}\)</span>
</h3>
<p>이제 <span class="math inline">\(\text{MORR}_{SU}\)</span>의 수식을 유도하여 보자. <span class="math inline">\(Y_{i}\)</span>를 <span class="math inline">\(i\)</span>th individual의 health status라 하고(case: 1, control: 0, <span class="math inline">\(1\le i \le n\)</span>), <span class="math inline">\(X_{i}\)</span>를 vector of covariates, <span class="math inline">\(G_i\)</span>를 <span class="math inline">\(i\)</span>th individual의 polygenic effect라 정의하자. 이제 polygenic model을 수식으로 나타내면 다음과 같다.</p>
<p><span class="math display">\[
\begin{aligned}
\text{Logit}[Pr(Y_{i}=1|X_{i},G_i)]=\beta_0+X_{i}'\beta_1+G_i
\end{aligned}
\]</span></p>
<p>(<span class="math inline">\(\beta_0\)</span>: intercept, <span class="math inline">\(\beta_1\)</span>: vector of fixed regression coefficients, <span class="math inline">\(G_i\)</span>: polygenic effect of <span class="math inline">\(i\)</span>th individual)</p>
<p>한편 <span class="math inline">\(G_i\)</span>들의 vector를 <span class="math inline">\(G=(G_1,G_2,\cdots,G_n)'\)</span>이라 하면, <span class="math inline">\(G\sim N(0,V_p\Sigma)\)</span> 이 된다(<span class="math inline">\(\Sigma\)</span>: genetic relationship matrix, <span class="math inline">\(V_p\)</span>: variance of polygenic effect).</p>
<p>이제 Conditional Odds를 계산해보면</p>
<p><span class="math display">\[
\begin{aligned}
\text{Odds}[Pr(Y_{i}=1|X_{i},G_i)]=\exp{(\beta_0)}\exp{(X_{i}'\beta_1)}\exp{(G_i)}
\end{aligned}
\]</span></p>
<p>로 표현할 수 있으며 임의로 뽑은 <span class="math inline">\(i\)</span>th individual과 <span class="math inline">\(j\)</span>th individual의 Odds ratio는 <span class="math inline">\(X\)</span>가 같을 때 다음과 같이 표현된다.</p>
<p><span class="math display">\[
\begin{aligned}
\frac{\text{Odds}[Pr(Y_{i}=1|X,G_i)]}{{\text{Odds}[Pr(Y_{j}=1|X,G_j)]}}=\exp{(G_i-G_j)}
\end{aligned}
\]</span></p>
<p>이제 두 가지 경우를 생각하자.</p>
<ol type="1">
<li><p><span class="math inline">\(i\)</span>와 <span class="math inline">\(j\)</span>를 unrelated individuals에서 뽑았을 경우이다. 이 때는 앞서 MOR과 마찬가지로 <span class="math inline">\((G_i-G_j) \sim N(0,2V_g)\)</span>가 된다.</p></li>
<li><p><span class="math inline">\(i\)</span>,<span class="math inline">\(j\)</span>를 sibling pair에서 뽑았다면, 즉 <span class="math inline">\(i\)</span>와 <span class="math inline">\(j\)</span>가 항상 sibling이라면 <span class="math inline">\(Cov(G_i,G_j)=\frac{1}{2}V_g\)</span> 이므로, <span class="math inline">\((G_i-G_j) \sim N(0,V_g)\)</span>가 된다.</p></li>
</ol>
<p>이제 unrelated individual과 sibling을 한 쌍씩 뽑아 각각 <span class="math inline">\(G_i, G_j\)</span>와 <span class="math inline">\(G_k, G_l\)</span>이라고 하면 Odds Ratio의 비인 OR Ratio(ORR)를 다음과 같이 정의한다.</p>
<p><span class="math display">\[
\begin{aligned}
\text{ORR}_{SU}=\frac{\text{OR}_{unrelated}}{\text{OR}_{sibling}}=\frac{\exp{|G_i-G_j|}}{\exp{|G_k-G_l|}} =  \exp{(|G_i-G_j|-|G_k-G_l|)}
\end{aligned}
\]</span></p>
<p><span class="math inline">\(|G_i-G_j|-|G_k-G_l|\)</span>는 <span class="math inline">\(|N(0,2V_g)| - |N(0, V_g)|\)</span> 의 분포를 따르는 것을 이용하여 median값을 계산하면 약 <span class="math inline">\(0.2453\times \sqrt{V_p}\)</span>이고 최종지표인 Median OR Ratio(MORR)은 아래와 같다(<strong>Appendix</strong>).</p>
<p><span class="math display">\[
\begin{aligned}
\text{MORR}_{SU}= \exp{(0.2453\times \sqrt{V_p})}
\end{aligned}
\]</span></p>
<p>즉 sibling pair들에서의 polygenic effect의 OR과 unrelated에서의 polygenic effect의 그것을 비교한 지표이며 범위는 1부터 무한대까지이다. <span class="math inline">\(\text{MORR}_{SU}\)</span>가 1이면 unrelated pair에서의 질병발생 양상의 차이가 sibling pair에서의 그것과 같게 되어 유전적인 부분이 전혀 없다고 해석할 수 있다. 값이 커질수록 sibling pair끼리는 unrelated pair끼리보다 질병발생 양상이 비슷하다고 볼 수 있으며 이는 곧 유전적인 부분이 큰 것으로 이해할 수 있다.</p>
</section></section><section id="apply-to-real-data" class="level2"><h2 class="anchored" data-anchor-id="apply-to-real-data">Apply to Real Data</h2>
<p>Healthy Twin Study, Korea의 데이터에 본 지표를 적용하였다<span class="citation" data-cites="sung2006we">(<a href="#ref-sung2006we" role="doc-biblioref">SUNG et al. 2006</a>)</span>. 가족-쌍둥이 구조로 이루어진 3,461명의 사람 중 지질검사와 음주, 흡연 정보가 있는 2,729명을 대상으로 150이상을 case, 150미만을 control로 정의하였으며 아무것도 보정하지 않은 Null model(Model 1), 성별과 연령을 보정한 모형(Model 2), 그리고 성별, 연령, 음주, 흡연력을 보정한 모형(Model 3)에 대해서 각각의 <span class="math inline">\(V_p\)</span>값과 그에 따른 <span class="math inline">\(\text{MORR}_{SU}\)</span>값을 제시하였다(Table 1). 분석은 <strong>R</strong> 3.5.1버전에서 <strong>hglm</strong> package를 이용하였다<span class="citation" data-cites="hglmpackage">(<a href="#ref-hglmpackage" role="doc-biblioref">Ronnegard, Shen, and Alam 2010</a>)</span>.</p>
<div class="cell">
<div class="cell-output-display">
<table class="table table-sm table-striped">
<caption>Variance parameter, heritablity and MORR of polygenic effect</caption>
<colgroup>
<col style="width: 26%">
<col style="width: 24%">
<col style="width: 24%">
<col style="width: 24%">
</colgroup>
<thead><tr class="header">
<th style="text-align: left;"></th>
<th style="text-align: center;">Model 1 (95%CI)</th>
<th style="text-align: center;">Model 2 (95%CI)</th>
<th style="text-align: center;">Model 3 (95%CI)</th>
</tr></thead>
<tbody>
<tr class="odd">
<td style="text-align: left;"><span class="math inline">\(V_p\)</span></td>
<td style="text-align: center;">0.59 (0.49-0.71)</td>
<td style="text-align: center;">0.68 (0.57-0.81)</td>
<td style="text-align: center;">0.66 (0.55-0.79)</td>
</tr>
<tr class="even">
<td style="text-align: left;">Heritablity</td>
<td style="text-align: center;">0.15 (0.13-0.18)</td>
<td style="text-align: center;">0.17 (0.15-0.2)</td>
<td style="text-align: center;">0.17 (0.14-0.19)</td>
</tr>
<tr class="odd">
<td style="text-align: left;"><span class="math inline">\(\text{MORR}_{su}\)</span></td>
<td style="text-align: center;">1.16 (1.13-1.19)</td>
<td style="text-align: center;">1.18 (1.15-1.22)</td>
<td style="text-align: center;">1.18 (1.14-1.21)</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>(Heritability: Intraclass correlation coefficients(<span class="math inline">\(\frac{V_p}{V_p+\frac{\pi^2}{3}}\)</span>), Model 1: no covariate, Model 2: age &amp; sex as covariates, Model 3: age, sex and alcohol/smoking status as covariates)</p>
<p>Model 2를 살펴보면 age와 sex의 effect를 보정하고 난 후의 <span class="math inline">\(V_p\)</span>값은 0.59(95% CI: 0.49-0.71)이고 이에 해당하는 <span class="math inline">\(\text{MORR}_{su}\)</span>의 값은 1.18(95% CI: 1.15-1.22)였으며 이는 sibling pair의 OR에 비해 unrelated pair의 OR의 값이 대략적으로 18% 더 높다고 해석할 수 있다. 한편 Null model(Model 1)의 <span class="math inline">\(\text{MORR}_{su}\)</span>의 값은 1.16(95% CI: 1.13-1.19)이며 age, sex, smoling status를 보정했을 때(Model 3)는 1.18(95% CI: 1.14-1.21)로 세 Model의 결과는 비슷했다. 이는 age, sex, alcohol, smoking status 보정 여부에 크게 상관없이 hypertriglycemia에 일정한 유전적인 영향이 존재함을 의미한다.</p>
</section><section id="conclusion" class="level2"><h2 class="anchored" data-anchor-id="conclusion">Conclusion</h2>
<p>저자가 제시한 <span class="math inline">\(\text{MORR}_{su}\)</span>의 개념을 이용하여 binary trait의 유전적인 정도를 직관적으로 설명할 수 있었다. 이는 해석이 어려운 heritability의 단점을 극복하였다는 의미가 있다. 또한 age, sex 등 다양한 covariates의 효과를 보정한 후의 유전적인 부분을 설명할 수 있고, 인구집단의 prevalence를 측정할 필요가 없다는 점에서 <span class="math inline">\(\lambda_s\)</span>의 단점을 보완한 지표라 할 수 있다.</p>
<p>향후 heritability와 <span class="math inline">\(\lambda_s\)</span> 각각의 장점은 살리고 단점은 보완한 이 지표가 binary trait의 polygenic effect를 직관적으로 설명하는 방법으로 널리 쓰이길 기대한다.</p>
</section><section id="appendix" class="level2"><h2 class="anchored" data-anchor-id="appendix">Appendix</h2>
<section id="calculate-morr" class="level3"><h3 class="anchored" data-anchor-id="calculate-morr">Calculate MORR</h3>
<p><span class="math inline">\(|N(0,2V_p)| - |N(0, V_p)|\)</span>의 median을 <span class="math inline">\(M\)</span>이라 하자. <strong>R</strong>의 <strong>distr</strong> package를 이용하면 아래와 같이 <span class="math inline">\(V_g\)</span>값들에 따른 <span class="math inline">\(M\)</span>값을 쉽게 계산할 수 있다<span class="citation" data-cites="distr">(<a href="#ref-distr" role="doc-biblioref">Ruckdeschel and Kohl 2014</a>)</span>.</p>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure"><p><img src="index_files/figure-html/unnamed-chunk-2-1.png" class="img-fluid figure-img" width="672"></p>
<p></p><figcaption class="figure-caption"><span class="math inline">\(V_p, \sqrt{V_p}\)</span> vs Median value of <span class="math inline">\(|N(0,2V_p)| - |N(0, V_p)|\)</span></figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>이 그래프의 직선의 기울기가 바로 0.2453이고 따라서 <span class="math inline">\(\text{MORR}_{SU}= \exp{(0.2453\times \sqrt{V_p})}\)</span>가 된다.</p>
</section><section id="compare-to-other-measures" class="level3"><h3 class="anchored" data-anchor-id="compare-to-other-measures">Compare to Other Measures</h3>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure"><p><img src="index_files/figure-html/unnamed-chunk-3-1.png" class="img-fluid figure-img" width="672"></p>
<p></p><figcaption class="figure-caption">Relationship between <span class="math inline">\(h^2\)</span> and <span class="math inline">\(\text{MORR}_{SU}\)</span></figcaption><p></p>
</figure>
</div>
</div>
</div>
<div class="cell">

</div>


<div id="refs" class="references csl-bib-body hanging-indent" role="doc-bibliography">
<div id="ref-1611.02910" class="csl-entry" role="doc-biblioentry">
Bonnet, Anna. 2016. <span>“Heritability Estimation of Diseases in Case-Control Studies.”</span> 2016. <a href="https://arxiv.org/abs/1611.02910">https://arxiv.org/abs/1611.02910</a>.
</div>
<div id="ref-browne2005variance" class="csl-entry" role="doc-biblioentry">
Browne, W. J., S. V. Subramanian, K. Jones, and H. Goldstein. 2005. <span>“Variance Partitioning in Multilevel Logistic Models That Exhibit Overdispersion.”</span> <em>Journal of the Royal Statistical Society: Series A (Statistics in Society)</em> 168 (3): 599–613. <a href="https://doi.org/10.1111/j.1467-985x.2004.00365.x">https://doi.org/10.1111/j.1467-985x.2004.00365.x</a>.
</div>
<div id="ref-Davies2639" class="csl-entry" role="doc-biblioentry">
Davies, Sarah W., Samuel V. Scarpino, Thanapat Pongwarin, James Scott, and Mikhail V. Matz. 2015. <span>“Estimating Trait Heritability in Highly Fecund Species.”</span> <em>G3: Genes, Genomes, Genetics</em> 5 (12): 2639–45. <a href="https://doi.org/10.1534/g3.115.020701">https://doi.org/10.1534/g3.115.020701</a>.
</div>
<div id="ref-guo1998inflation" class="csl-entry" role="doc-biblioentry">
Guo, Sun-Wei. 1998. <span>“Inflation of Sibling Recurrence-Risk Ratio, Due to Ascertainment Bias and/or Overreporting.”</span> <em>The American Journal of Human Genetics</em> 63 (1): 252–58. <a href="https://doi.org/10.1086/301928">https://doi.org/10.1086/301928</a>.
</div>
<div id="ref-guo2002sibling" class="csl-entry" role="doc-biblioentry">
———. 2002. <span>“Sibling Recurrence Risk Ratio as a Measure of Genetic Effect: Caveat Emptor!”</span> <em>The American Journal of Human Genetics</em> 70 (3): 818–19. <a href="https://doi.org/10.1086/339369">https://doi.org/10.1086/339369</a>.
</div>
<div id="ref-guo2000familial" class="csl-entry" role="doc-biblioentry">
Guo, S.-W. 2000. <span>“Familial Aggregation of Environmental Risk Factors and Familial Aggregation of Disease.”</span> <em>American Journal of Epidemiology</em> 151 (11): 1121–31. <a href="https://doi.org/10.1093/oxfordjournals.aje.a010156">https://doi.org/10.1093/oxfordjournals.aje.a010156</a>.
</div>
<div id="ref-larsen2005appropriate" class="csl-entry" role="doc-biblioentry">
Larsen, Klaus, and Juan Merlo. 2005. <span>“Appropriate Assessment of Neighborhood Effects on Individual Health: Integrating Random and Fixed Effects in Multilevel Logistic Regression.”</span> <em>American Journal of Epidemiology</em> 161 (1): 81–88. <a href="https://doi.org/10.1093/aje/kwi017">https://doi.org/10.1093/aje/kwi017</a>.
</div>
<div id="ref-larsen2000interpreting" class="csl-entry" role="doc-biblioentry">
Larsen, Klaus, Jørgen Holm Petersen, Esben Budtz-Jørgensen, and Lars Endahl. 2000. <span>“Interpreting Parameters in the Logistic Regression Model with Random Effects.”</span> <em>Biometrics</em> 56 (3): 909–14. <a href="https://doi.org/10.1111/j.0006-341X.2000.00909.x">https://doi.org/10.1111/j.0006-341X.2000.00909.x</a>.
</div>
<div id="ref-lee1996hierarchical" class="csl-entry" role="doc-biblioentry">
Lee, Y., and J. A. Nelder. 1996. <span>“Hierarchical Generalized Linear Models.”</span> <em>Journal of the Royal Statistical Society. Series B (Methodological)</em> 58 (4): 619–78. <a href="http://www.jstor.org/stable/2346105">http://www.jstor.org/stable/2346105</a>.
</div>
<div id="ref-manolio2009finding" class="csl-entry" role="doc-biblioentry">
Manolio, Teri A., Francis S. Collins, Nancy J. Cox, David B. Goldstein, Lucia A. Hindorff, David J. Hunter, Mark I. McCarthy, et al. 2009. <span>“Finding the Missing Heritability of Complex Diseases.”</span> <em>Nature</em> 461 (7265): 747–53. <a href="https://doi.org/10.1038/nature08494">https://doi.org/10.1038/nature08494</a>.
</div>
<div id="ref-merlo2006brief" class="csl-entry" role="doc-biblioentry">
Merlo, Juan, Basile Chaix, Henrik Ohlsson, Anders Beckman, Kristina Johnell, Per Hjerpe, L Råstam, and K Larsen. 2006. <span>“A Brief Conceptual Tutorial of Multilevel Analysis in Social Epidemiology: Using Measures of Clustering in Multilevel Logistic Regression to Investigate Contextual Phenomena.”</span> <em>Journal of Epidemiology &amp; Community Health</em> 60 (4): 290–97. <a href="https://doi.org/10.1136/jech.2004.029454">https://doi.org/10.1136/jech.2004.029454</a>.
</div>
<div id="ref-hglmpackage" class="csl-entry" role="doc-biblioentry">
Ronnegard, Lars, Xia Shen, and Moudud Alam. 2010. <span>“Hglm: A Package for Fitting Hierarchical Generalized Linear Models.”</span> <em>The R Journal</em> 2 (2): 20–28. <a href="http://journal.r-project.org/archive/2010-2/RJournal_2010-2_Roennegaard~et~al.pdf">http://journal.r-project.org/archive/2010-2/RJournal_2010-2_Roennegaard~et~al.pdf</a>.
</div>
<div id="ref-distr" class="csl-entry" role="doc-biblioentry">
Ruckdeschel, Peter, and Matthias Kohl. 2014. <span>“General Purpose Convolution Algorithm in S4 Classes by Means of FFT.”</span> <em>Journal of Statistical Software, Articles</em> 59 (4): 1–25. <a href="https://doi.org/10.18637/jss.v059.i04">https://doi.org/10.18637/jss.v059.i04</a>.
</div>
<div id="ref-rybicki2000relationship" class="csl-entry" role="doc-biblioentry">
Rybicki, Benjamin A., and Robert C. Elston. 2000. <span>“The Relationship Between the Sibling Recurrence-Risk Ratio and Genotype Relative Risk.”</span> <em>The American Journal of Human Genetics</em> 66 (2): 593–604. <a href="https://doi.org/10.1086/302778">https://doi.org/10.1086/302778</a>.
</div>
<div id="ref-sung2006we" class="csl-entry" role="doc-biblioentry">
SUNG, JOOHON, SUNG-IL CHO, YUN-MI SONG, KAYOUNG LEE, EUN-YOUNG CHOI, MINA HA, JIHAE KIM, et al. 2006. <span>“Do We Need More Twin Studies? The Healthy Twin Study, Korea.”</span> <em>International Journal of Epidemiology</em> 35 (2): 488–90. <a href="https://doi.org/10.1093/ije/dyi294">https://doi.org/10.1093/ije/dyi294</a>.
</div>
<div id="ref-Tenesa2013" class="csl-entry" role="doc-biblioentry">
Tenesa, Albert, and Chris S. Haley. 2013. <span>“The Heritability of Human Disease: Estimation, Uses and Abuses.”</span> <em>Nature Reviews Genetics</em> 14 (2): 139–49. <a href="https://doi.org/10.1038/nrg3377">https://doi.org/10.1038/nrg3377</a>.
</div>
<div id="ref-vattikuti2012heritability" class="csl-entry" role="doc-biblioentry">
Vattikuti, Juen AND Chow, Shashaank AND Guo. 2012. <span>“Heritability and Genetic Correlations Explained by Common SNPs for Metabolic Syndrome Traits.”</span> <em>PLOS Genetics</em> 8 (3): 1–8. <a href="https://doi.org/10.1371/journal.pgen.1002637">https://doi.org/10.1371/journal.pgen.1002637</a>.
</div>
<div id="ref-vigre2004intr" class="csl-entry" role="doc-biblioentry">
Vigre, Håkan, Ian R. Dohoo, Henrik Stryhn, and Marie Erika Busch. 2004. <span>“Intra-Unit Correlations in Seroconversion to Actinobacillus Pleuropneumoniae and Mycoplasma Hyopneumoniae at Different Levels in Danish Multi-Site Pig Production Facilities.”</span> <em>Preventive Veterinary Medicine</em> 63 (1): 9–28. https://doi.org/<a href="https://doi.org/10.1016/j.prevetmed.2004.02.002">https://doi.org/10.1016/j.prevetmed.2004.02.002</a>.
</div>
<div id="ref-1611.02910" class="csl-entry" role="doc-biblioentry">
Bonnet, Anna. 2016. <span>“Heritability Estimation of Diseases in Case-Control Studies.”</span> 2016. <a href="https://arxiv.org/abs/1611.02910">https://arxiv.org/abs/1611.02910</a>.
</div>
<div id="ref-browne2005variance" class="csl-entry" role="doc-biblioentry">
Browne, W. J., S. V. Subramanian, K. Jones, and H. Goldstein. 2005. <span>“Variance Partitioning in Multilevel Logistic Models That Exhibit Overdispersion.”</span> <em>Journal of the Royal Statistical Society: Series A (Statistics in Society)</em> 168 (3): 599–613. <a href="https://doi.org/10.1111/j.1467-985x.2004.00365.x">https://doi.org/10.1111/j.1467-985x.2004.00365.x</a>.
</div>
<div id="ref-Davies2639" class="csl-entry" role="doc-biblioentry">
Davies, Sarah W., Samuel V. Scarpino, Thanapat Pongwarin, James Scott, and Mikhail V. Matz. 2015. <span>“Estimating Trait Heritability in Highly Fecund Species.”</span> <em>G3: Genes, Genomes, Genetics</em> 5 (12): 2639–45. <a href="https://doi.org/10.1534/g3.115.020701">https://doi.org/10.1534/g3.115.020701</a>.
</div>
<div id="ref-guo1998inflation" class="csl-entry" role="doc-biblioentry">
Guo, Sun-Wei. 1998. <span>“Inflation of Sibling Recurrence-Risk Ratio, Due to Ascertainment Bias and/or Overreporting.”</span> <em>The American Journal of Human Genetics</em> 63 (1): 252–58. <a href="https://doi.org/10.1086/301928">https://doi.org/10.1086/301928</a>.
</div>
<div id="ref-guo2002sibling" class="csl-entry" role="doc-biblioentry">
———. 2002. <span>“Sibling Recurrence Risk Ratio as a Measure of Genetic Effect: Caveat Emptor!”</span> <em>The American Journal of Human Genetics</em> 70 (3): 818–19. <a href="https://doi.org/10.1086/339369">https://doi.org/10.1086/339369</a>.
</div>
<div id="ref-guo2000familial" class="csl-entry" role="doc-biblioentry">
Guo, S.-W. 2000. <span>“Familial Aggregation of Environmental Risk Factors and Familial Aggregation of Disease.”</span> <em>American Journal of Epidemiology</em> 151 (11): 1121–31. <a href="https://doi.org/10.1093/oxfordjournals.aje.a010156">https://doi.org/10.1093/oxfordjournals.aje.a010156</a>.
</div>
<div id="ref-larsen2005appropriate" class="csl-entry" role="doc-biblioentry">
Larsen, Klaus, and Juan Merlo. 2005. <span>“Appropriate Assessment of Neighborhood Effects on Individual Health: Integrating Random and Fixed Effects in Multilevel Logistic Regression.”</span> <em>American Journal of Epidemiology</em> 161 (1): 81–88. <a href="https://doi.org/10.1093/aje/kwi017">https://doi.org/10.1093/aje/kwi017</a>.
</div>
<div id="ref-larsen2000interpreting" class="csl-entry" role="doc-biblioentry">
Larsen, Klaus, Jørgen Holm Petersen, Esben Budtz-Jørgensen, and Lars Endahl. 2000. <span>“Interpreting Parameters in the Logistic Regression Model with Random Effects.”</span> <em>Biometrics</em> 56 (3): 909–14. <a href="https://doi.org/10.1111/j.0006-341X.2000.00909.x">https://doi.org/10.1111/j.0006-341X.2000.00909.x</a>.
</div>
<div id="ref-lee1996hierarchical" class="csl-entry" role="doc-biblioentry">
Lee, Y., and J. A. Nelder. 1996. <span>“Hierarchical Generalized Linear Models.”</span> <em>Journal of the Royal Statistical Society. Series B (Methodological)</em> 58 (4): 619–78. <a href="http://www.jstor.org/stable/2346105">http://www.jstor.org/stable/2346105</a>.
</div>
<div id="ref-manolio2009finding" class="csl-entry" role="doc-biblioentry">
Manolio, Teri A., Francis S. Collins, Nancy J. Cox, David B. Goldstein, Lucia A. Hindorff, David J. Hunter, Mark I. McCarthy, et al. 2009. <span>“Finding the Missing Heritability of Complex Diseases.”</span> <em>Nature</em> 461 (7265): 747–53. <a href="https://doi.org/10.1038/nature08494">https://doi.org/10.1038/nature08494</a>.
</div>
<div id="ref-merlo2006brief" class="csl-entry" role="doc-biblioentry">
Merlo, Juan, Basile Chaix, Henrik Ohlsson, Anders Beckman, Kristina Johnell, Per Hjerpe, L Råstam, and K Larsen. 2006. <span>“A Brief Conceptual Tutorial of Multilevel Analysis in Social Epidemiology: Using Measures of Clustering in Multilevel Logistic Regression to Investigate Contextual Phenomena.”</span> <em>Journal of Epidemiology &amp; Community Health</em> 60 (4): 290–97. <a href="https://doi.org/10.1136/jech.2004.029454">https://doi.org/10.1136/jech.2004.029454</a>.
</div>
<div id="ref-hglmpackage" class="csl-entry" role="doc-biblioentry">
Ronnegard, Lars, Xia Shen, and Moudud Alam. 2010. <span>“Hglm: A Package for Fitting Hierarchical Generalized Linear Models.”</span> <em>The R Journal</em> 2 (2): 20–28. <a href="http://journal.r-project.org/archive/2010-2/RJournal_2010-2_Roennegaard~et~al.pdf">http://journal.r-project.org/archive/2010-2/RJournal_2010-2_Roennegaard~et~al.pdf</a>.
</div>
<div id="ref-distr" class="csl-entry" role="doc-biblioentry">
Ruckdeschel, Peter, and Matthias Kohl. 2014. <span>“General Purpose Convolution Algorithm in S4 Classes by Means of FFT.”</span> <em>Journal of Statistical Software, Articles</em> 59 (4): 1–25. <a href="https://doi.org/10.18637/jss.v059.i04">https://doi.org/10.18637/jss.v059.i04</a>.
</div>
<div id="ref-rybicki2000relationship" class="csl-entry" role="doc-biblioentry">
Rybicki, Benjamin A., and Robert C. Elston. 2000. <span>“The Relationship Between the Sibling Recurrence-Risk Ratio and Genotype Relative Risk.”</span> <em>The American Journal of Human Genetics</em> 66 (2): 593–604. <a href="https://doi.org/10.1086/302778">https://doi.org/10.1086/302778</a>.
</div>
<div id="ref-sung2006we" class="csl-entry" role="doc-biblioentry">
SUNG, JOOHON, SUNG-IL CHO, YUN-MI SONG, KAYOUNG LEE, EUN-YOUNG CHOI, MINA HA, JIHAE KIM, et al. 2006. <span>“Do We Need More Twin Studies? The Healthy Twin Study, Korea.”</span> <em>International Journal of Epidemiology</em> 35 (2): 488–90. <a href="https://doi.org/10.1093/ije/dyi294">https://doi.org/10.1093/ije/dyi294</a>.
</div>
<div id="ref-Tenesa2013" class="csl-entry" role="doc-biblioentry">
Tenesa, Albert, and Chris S. Haley. 2013. <span>“The Heritability of Human Disease: Estimation, Uses and Abuses.”</span> <em>Nature Reviews Genetics</em> 14 (2): 139–49. <a href="https://doi.org/10.1038/nrg3377">https://doi.org/10.1038/nrg3377</a>.
</div>
<div id="ref-vattikuti2012heritability" class="csl-entry" role="doc-biblioentry">
Vattikuti, Juen AND Chow, Shashaank AND Guo. 2012. <span>“Heritability and Genetic Correlations Explained by Common SNPs for Metabolic Syndrome Traits.”</span> <em>PLOS Genetics</em> 8 (3): 1–8. <a href="https://doi.org/10.1371/journal.pgen.1002637">https://doi.org/10.1371/journal.pgen.1002637</a>.
</div>
<div id="ref-vigre2004intr" class="csl-entry" role="doc-biblioentry">
Vigre, Håkan, Ian R. Dohoo, Henrik Stryhn, and Marie Erika Busch. 2004. <span>“Intra-Unit Correlations in Seroconversion to Actinobacillus Pleuropneumoniae and Mycoplasma Hyopneumoniae at Different Levels in Danish Multi-Site Pig Production Facilities.”</span> <em>Preventive Veterinary Medicine</em> 63 (1): 9–28. https://doi.org/<a href="https://doi.org/10.1016/j.prevetmed.2004.02.002">https://doi.org/10.1016/j.prevetmed.2004.02.002</a>.
</div>
</div>
</section></section><div id="quarto-appendix" class="default"><section class="quarto-appendix-contents"><h2 class="anchored quarto-appendix-heading">Citation</h2><div><div class="quarto-appendix-secondary-label">BibTeX citation:</div><pre class="sourceCode code-with-copy quarto-appendix-bibtex"><code class="sourceCode bibtex">@online{kim2018,
  author = {Jinseob Kim},
  title = {New {Scale} {Measure} of {Heritability} in {Binary} {Trait}
    {Using} {Median} {Odds} {Ratio:} {Median} {OR} {Ratio}},
  date = {11/08/2018},
  url = {https://blog.zarathu.com/posts/2018-11-08-medianorratio},
  langid = {en}
}
</code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre><div class="quarto-appendix-secondary-label">For attribution, please cite this work as:</div><div id="ref-kim2018" class="csl-entry quarto-appendix-citeas" role="doc-biblioentry">
Jinseob Kim. 11AD–8AD. <span>“New Scale Measure of Heritability in
Binary Trait Using Median Odds Ratio: Median OR Ratio.”</span> 11AD–8AD.
<a href="https://blog.zarathu.com/posts/2018-11-08-medianorratio">https://blog.zarathu.com/posts/2018-11-08-medianorratio</a>.
</div></div></section></div></main><!-- /main --><script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    target: function(trigger) {
      return trigger.previousElementSibling;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    setTimeout(function() {
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      let href = ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const cites = ref.parentNode.getAttribute('data-cites').split(' ');
    tippyHover(ref, function() {
      var popup = window.document.createElement('div');
      cites.forEach(function(cite) {
        var citeDiv = window.document.createElement('div');
        citeDiv.classList.add('hanging-indent');
        citeDiv.classList.add('csl-entry');
        var biblioDiv = window.document.getElementById('ref-' + cite);
        if (biblioDiv) {
          citeDiv.innerHTML = biblioDiv.innerHTML;
        }
        popup.appendChild(citeDiv);
      });
      return popup.innerHTML;
    });
  }
});
</script><script src="https://giscus.app/client.js" data-repo="zarathucorp/giscus-blog" data-repo-id="R_kgDOHztuxg" data-category="General" data-category-id="DIC_kwDOHztuxs4CQ6h5" data-mapping="title" data-reactions-enabled="1" data-emit-metadata="0" data-input-position="top" data-theme="light" data-lang="en" crossorigin="anonymous" async="">
</script>
</div> <!-- /content -->
<footer class="footer"><div class="nav-footer">
      <div class="nav-footer-center">
        <ul class="footer-items list-unstyled">
<li class="nav-item">
    <a class="nav-link" href="https://www.zarathu.com">© Copyright 2019 Zarathu Co.,Ltd.</a>
  </li>  
    <li class="nav-item">
    <a class="nav-link" href="https://raw.githubusercontent.com/zarathucorp/blog/master/LICENSE">Software licensed under the MIT License.</a>
  </li>  
</ul>
</div>
  </div>
</footer>


<script src="../../site_libs/quarto-html/zenscroll-min.js"></script>
</body></html>